test/transcoding/SPV_INTEL_memory_access_aliasing/intel-alias-lifetime.ll

```diff
@@ -0,0 +1,30 @@
+; The test checks if the translator won't crash
+; TODO need to figure out how to translate the alias.scope/noalias metadata
+; in a case when it attached to a call to lifetime intrinsic
```

Comment:
How are they translated now?

Reply:
lifetime start/stop are mapped on OpLifetimeStart/Stop. alias.scope/noalias metadata attached to these instructions are currently being dropped.